airflow/settings.py

```diff
@@ -261,7 +261,7 @@ def configure_action_logging():
 except Exception:
     pass
 
-configure_logging()
+logging_class_path = configure_logging()
```

Comment:
Do we need to assign this to a variable ??

Reply:
Ya the logging_class_path is used [here](https://github.com/apache/incubator-airflow/pull/4253/files#diff-ced4fd65ce02db58eed692eef6e01d05R546). Just trying to keep it consistent between two usage.

Reply:
👍